frontend/src/util/md5.ts

```diff
@@ -0,0 +1,170 @@
+function md5cycle(x, k) {
+    var a = x[0], b = x[1], c = x[2], d = x[3];
```

Comment:
Whats this? 💖
![giphy](https://user-images.githubusercontent.com/14180997/74653612-74796d00-5180-11ea-9360-9ee625db6231.gif)

Reply:
![giphy](https://media.giphy.com/media/3oFzmkkwfOGlzZ0gxi/giphy.gif)